### ISSUE TITLE
Actual prediction

### ISSUE THREAD

**Issue by europa502** [kimanalytics/Recurrent-Neural-Network-to-Predict-Stock-Prices#1]
As far as I've understood from this program is that yo r predicting the stock value on the input stock value. Thus here you r inputting a stock value of a particular day and the model is throwing out the predicted stock value of the same day. How can I modify it to predict the value of the next day? 
At first I thought maybe by putting x_train as as the no. of days(from day 0 to day N) and y_train as the opening stock value, but then i get huge loss values (more than 200000).

**Comment by soymintc** [comment]
I agree. It'd be better if you make an alternative testing scheme to see the differences of the future-predicted stock price instead of simply plugging in the "inputs" variable obtained from real_stock_price. 
As of now, the code only seems to predict the stock price of 2017 USING the stock price data of 2017.

**Comment by MachineTrader** [comment]
Agree
